neo.UnitTests/Ledger/UT_MemoryPool.cs

```diff
@@ -171,6 +208,36 @@ public void BlockPersistMovesTxToUnverifiedAndReverification()
             _unit.UnverifiedSortedTxCount.Should().Be(0);
         }
 
+        [TestMethod]
+        public void BlockPersistAndReverificationWillAbandonTxAsBalanceTransfered()
```

Comment:
I reverted the `Memory Pool` changes and this UT pass, so it not prove the specific case.

Reply:
I got, I should fix the balance, update later

Reply:
done